test/analytics/tests/instrumented_events/from_the_browser/loaded_kibana.ts

```diff
@@ -19,36 +19,57 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await common.navigateToApp('home');
     });
 
-    it('should emit the "Loaded Kibana" event', async () => {
+    it('should emit the legacy "Loaded Kibana"', async () => {
       const [event] = await ebtUIHelper.getEvents(1, { eventTypes: ['Loaded Kibana'] });
+
       expect(event.event_type).to.eql('Loaded Kibana');
       expect(event.properties).to.have.property('kibana_version');
       expect(event.properties.kibana_version).to.be.a('string');
       expect(event.properties).to.have.property('protocol');
       expect(event.properties.protocol).to.be.a('string');
+    });
+
+    it('should emit the new kibana-loaded events', async () => {
```

Comment:
Nit: "new" is relative. Once we delete the "legacy" event, the term "new" becomes confusing. Also, the event name is `kibana_loaded` (with an underscore). I suggest:

```suggestion
    it('should emit the kibana_loaded event', async () => {
```